src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  const totalCost = days * dailyRate;
+  const smallDiscount = 20;
+  const discount = 50;
+  let cost = 0;
```

Comment:
The variable `cost` is unnecessary because you can return the result directly within the `if` blocks. This will simplify the function and adhere to the checklist's preference for using `const` over `let` where possible.